config/crd/bases/keda.sh_scaledjobs.yaml

```diff
@@ -8220,6 +8220,8 @@ spec:
           status:
             description: ScaledJobStatus defines the observed state of ScaledJob
             properties:
+              Paused:
```

Comment:
We already support pausing in Scaled Object, can you ensure we are using the same approach please?

Reply:
> My understanding is to pause a ScaledObject is to set the replica count via the annotation autoscaling.keda.sh/paused-replicas: "n" where n = 0 scales down workload and pauses autoscaling.

This will change where they can also just pause in the future, but you have a point.

My suggestion was to align on the naming etc but this does not make much sense for jobs indeed. Agreed @JorTurFer @zroubalik ?

Reply:
I see @flux-benj's point and I agree. We present the ScaledJob as a mechanism for running long term tasks and killing them it's too agressive IMO. I see the `"autoscaling.keda.sh/paused"` (or maybe `"autoscaling.keda.sh/paused-job"`)  annotation as way to not create more jobs but keeping already created jobs alive until the finish by themselves

Reply:
Agreed, but let's stick with `"autoscaling.keda.sh/paused"` though since SO will have this in the future as well